sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/joins.scala

```diff
@@ -484,6 +484,20 @@ trait JoinSelectionHelper {
     hint.rightHint.exists(_.strategy.contains(NO_BROADCAST_AND_REPLICATION))
   }
 
+  def canLeftSideBeBroadcast(left: LogicalPlan, conf: SQLConf,
+                             hint: JoinHint, joinType: JoinType): Boolean = {
```

Comment:
Apologies, didn't see this earlier. Please indent in similar way. Parameters starting in new line indented by 4 spaces.